maximbircu/devtools-library
# Configure SwiftLint

Closes: 


### What has been done
 1. Added `.swiftlint.yml` configuration to include only the source file, thus excluding linting the dependencies. Was done for `devtools/ios` and `samples/ios`.
 2. Was updated the build phase command of SwiftLint to included the configuration file flag.
 3. Added Gemfile for `devtools/ios` and `samples/ios` with `cocoapods`. This will make sure that everyone (including CI) will use the same version of Ruby dependencies. Now, to install dependencies for iOS you have to run: `bundle exec pod install`.

### How to test
1.  In `devtools/ios` run: `bundle install && bundle exec pod install`. Open the Xcode project (`xed .`), and build the project. No SwiftLint warnings should appear.
2. In `samples/ios` run: `bundle install && bundle exec pod install`. Open the Xcode project (`xed .`), and build the project. No SwiftLint warnings should appear.
